controllers/homeController.js

```diff
@@ -76,6 +87,31 @@ var findCommonChampions = function(championsA, championsB) {
     return commonChampions
 }
 
+var compareCommonChampions = function(commonChampions) {
+    var aCount = 0;
+    var bCount = 0;
+    commonChampions.forEach(function(championTuple) {
```

Comment:
i was thinking a little different for this, youd map your champion array to, for now an array of booleans and then you can traverse that to figure out who is better. This is the route i was imagining wed take in the future, it'd evolve to like returning a list of objects and the object can contain whatever we want about the comparison we ran on the two champs.